src/rbac/defaultRoleManager.ts

```diff
@@ -333,8 +349,11 @@ export class DefaultRoleManager implements RoleManager {
     if (!allRoles.hasRole(name, this.matchingFunc)) {
       return [];
     }
-
-    return [...allRoles.values()].filter((n) => n.hasDirectRole(name)).map((n) => n.name);
+    const users = [];
```

Comment:
I felt that the previous implementation was a little bit confusing and has the possibility to process a long list 3 times, while this does it only once.

Reply:
for loop is faster :+1: